Formula/aws-sam-cli.rb

```diff
@@ -1,10 +1,11 @@
+# -*- coding: utf-8 -*-
```

Comment:
why do we need this?

Reply:
It's being auto-added by my editor - I don't think it's strictly needed, but it's good form when your Ruby file contains Unicode characters, as this one does. I'd actually recommend keeping it.

Reply:
cool, as long as its best practice and the formula is still tap-able. I'm okay with it.